shanoir-ng-front/src/app/shared/side-menu/side-menu.component.html

```diff
@@ -28,8 +28,24 @@
 </div>
 
 <ul class="menu-list">
-
     <li routerLink="/" routerLinkActive="active"><i class="fas fa-home"></i>Home</li>
+    <li (click)="tasksOpened = !tasksOpened" class="notifications">
+        <i class="fas fa-tasks"></i>Tasks
+        <span *ngIf="notificationsService.nbProcess > 0" class="notif">( {{notificationsService.nbProcess}} pending - {{notificationsService.totalProgress() * 100 | number:'0.0-0'}}%)</span>
+        <!-- <button class="awesome" (click)="notificationsService.refresh()" >
+            <i class="fas fa-sync"></i>
+        </button> -->
+        <span class="chevron"><i class="fas fa-chevron-down"></i></span>
+    </li>
+    <div *ngIf="tasksOpened" class="sub-menu" [@slideDown]="dataOpened">
+        <li routerLink="/task" routerLinkActive="active"><i class="fas fa-tasks"></i>Manage my tasks</li>
+        <li *ngFor="let item of notificationsService.tasksInProgress" class="single-task">
+            <i class="fas fa-chevron-right"></i>

```

Comment:
This chevron make it looks like we can expand it, which is not the case (I guess ?)
Maybe put a dot or something ? :)